### PR TITLE
opencascade: update to 7.5.3

### PR DESCRIPTION
**Pull request by MehdiChinoune**

### CODE REVIEW

File: mingw-w64-opencascade/PKGBUILD [hunk 1]
```diff
@@ -137,17 +143,21 @@ build() {
...
   cd "${srcdir}/build-${MSYSTEM}-static"
-  DESTDIR=${pkgdir} cmake --build . --target install
+  DESTDIR=${pkgdir} cmake --install .
 
   #Shared Install
   cd "${srcdir}/build-${MSYSTEM}-shared"
-  DESTDIR=${pkgdir} cmake --build . --target install
+  DESTDIR=${pkgdir} cmake --install .
 
   install -Dm644 ${srcdir}/occt-${_pkgver2}/LICENSE_LGPL_21.txt ${pkgdir}${MINGW_PREFIX}/share/licenses/${_realname}/LICENSE
```
Comment (chennes) [inline mingw-w64-opencascade/PKGBUILD:158]: This line fails if there is a space in the ${srcdir} directory path.

Comment (MehdiChinoune) [reply mingw-w64-opencascade/PKGBUILD]: Could you test the resulted packages from https://github.com/msys2/MINGW-packages/actions/runs/1227144810 (artifacts)

**Comment by chennes** [comment]
I'm still getting two errors with this PR, though I believe that both pre-date the PR and exist in the current version as well. 

First, the installation of the license file needs quotes around the entire paths to handle cases where the path has a space in it:
```
install -Dm644 "${srcdir}/occt-${_pkgver2}/LICENSE_LGPL_21.txt" "${pkgdir}${MINGW_PREFIX}/share/licenses/${_realname}/LICENSE"
```

Second, it looks like I'm getting a strange GitHub-related path when using the CI artifacts. The error I receive when trying to compile against this version of OpenCASCADE is:
```
ninja: error: 'D:/a/_temp/msys/msys64/mingw64/lib/libopenvr_api.dll.a', needed by 'bin/libSMDS.dll', missing and no known rule to make it
```
That "D:/a/_temp/" looks like the CI path here, it doesn't exist in any form on my system. If I compile this PR from scratch on my own system, it works fine and the error does not occur.

**Comment by MehdiChinoune** [comment]
> I'm still getting two errors with this PR, though I believe that both pre-date the PR and exist in the current version as well.
> 
> First, the installation of the license file needs quotes around the entire paths to handle cases where the path has a space in it:
> 
> ```
> install -Dm644 "${srcdir}/occt-${_pkgver2}/LICENSE_LGPL_21.txt" "${pkgdir}${MINGW_PREFIX}/share/licenses/${_realname}/LICENSE"
> ```

As you can see, It was like that before I tried to update it, It was never recommended to use folders with space character for development. There are many packages in this repo that do not quotes their paths and this the first time I hear someone complaining about it. Maybe you should rename your folders.

> Second, it looks like I'm getting a strange GitHub-related path when using the CI artifacts. The error I receive when trying to compile against this version of OpenCASCADE is:
> 
> ```
> ninja: error: 'D:/a/_temp/msys/msys64/mingw64/lib/libopenvr_api.dll.a', needed by 'bin/libSMDS.dll', missing and no known rule to make it
> ```
> 
> That "D:/a/_temp/" looks like the CI path here, it doesn't exist in any form on my system. If I compile this PR from scratch on my own system, it works fine and the error does not occur.

It looks like cmake files need to fixed, I don't have much time to investigate it, feel free to open another pr.

**Pull request closed without merging**